.github/workflows/compile_changelogs.yml

```diff
@@ -0,0 +1,36 @@
+name: Compile changelogs
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  compile:
+    name: "Compile changelogs"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Setup python"
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: "Install deps"
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pyyaml
+      - name: "Checkout"
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 25
+      - name: "Compile"
+        run: |
+          python tools/ss13_genchangelog.py html/changelog.html html/changelogs
+      - name: Commit
+        run: |
+          git config --local user.email "action@github.com"
```

Comment:
```suggestion
          git config --local user.email "tgstation-server@users.noreply.github.com"
```

Reply:
disagree, this is not by that user

Reply:
Yeah this is a github action, using that email is correct